### PR TITLE
LG 01-06 add data, not just data models

### PR DESCRIPTION
**Pull request by gernotstarke**
see #660

### CODE REVIEW

File: docs/01-basics/LG-01-06.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 // tag::DE[]
 [[LG-01-06]]
-==== LZ 01-06: Bedeutung und Entwurf von Datenmodellen (R2)
+==== LZ 01-06: Bedeutung und Entwurf von Daten und Datenmodellen (R2)
 
-Softwarearchitekt:innen verstehen die Bedeutung von Datenmodellen für die Architektur.  
+Softwarearchitekt:innen verstehen die Bedeutung von Data and Datenmodellen für die Architektur.
```
Comment (alxlo) [inline docs/01-basics/LG-01-06.adoc:6]:
```suggestion
Softwarearchitekt:innen verstehen die Bedeutung von Daten and Datenmodellen für die Architektur.  
```

**Review by alxlo — DISMISSED** [review]
Last iteration is fine by me. @skogsbaer what do you think?

**Review by sippsack — APPROVED** [review]

**Comment by skogsbaer** [comment]
Looks good!

**Comment by ulibecker** [comment]
Looks good to me

**Pull request merged**